modules/porous_flow/test/tests/hysteresis/hys_order_01.i

```diff
@@ -0,0 +1,154 @@
+# Test that PorousFlowHysteresisOrder correctly calculates hysteresis order
+# Water is removed from the system (so order = 0) until saturation = S0
+# Then, water is added to the system (so order = 1) until saturation = S1
+# Then, water is removed from the system (so order = 2)
+# More water is removed from the system so that the saturation < S0 (so order = 0)
+[Mesh]
```

Comment:
Minor, but probably should be GeneratedMeshGenerators now

Reply:
done